pkgs/servers/etebase/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib, python3, fetchFromGitHub }:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_3;
+    };
+  };
+in
+  with py.pkgs;
+
+buildPythonApplication rec {
+  pname = "etebase-server";
+  version = "0.5.3";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "etesync";
+    repo = "server";
+    rev = "v${version}";
+    sha256 = "03bhdj7xviwcfaz5449psighdykgizhnv4n0298f6mgd4b5bwrp6";
+  };
+
+  patches = [ ./secret.patch ];
+
+  propagatedBuildInputs = with pythonPackages; [
+    asgiref
+    cffi
+    django
+    django-cors-headers
+    djangorestframework
+    drf-nested-routers
+    msgpack
+    psycopg2
+    pycparser
+    pynacl
+    pytz
+    six
+    sqlparse
+  ];
+
+  installPhase = ''
+    mkdir -p "$out/bin" "$out/lib"
+    mkdir -p "$out/share/doc/${pname}"
+    mkdir -p "$out/share/licenses/${pname}"
+
+    install -Dm644 "README.md" "$out/share/doc/${pname}/README.md"
+    mv "example-configs" "$out/share/doc/${pname}/"
```

Comment:
unsure if everything should be copied as-is, maybe it would be nice to not keep irrelevant things, but no strong feelings on it

Reply:
I'll look into it after I get the service running. Thanks for your feedback so far 🙂.